.travis.yml

```diff
@@ -24,12 +22,12 @@ install:
  - cabal --version
  - echo "$(ghc --version) [$(ghc --print-project-git-commit-id 2> /dev/null || echo '?')]"
  - travis_retry cabal update
- - cabal install --only-dependencies --enable-tests --enable-benchmarks
+ - cabal install --only-dependencies
 
 # Here starts the actual work to be performed for the package under test; any command which exits with a non-zero exit code causes the build to fail.
 script:
  - if [ -f configure.ac ]; then autoreconf -i; fi
- - cabal configure --enable-tests --enable-benchmarks -v2 --disable-shared # -v2 provides useful information for debugging
```

Comment:
`--enable-tests --enable-benchmarks` was causing errors on HEAD. I think it's fine to just remove it—this package doesn't have tests or benchmarks anyway